ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -72,12 +74,13 @@ void RSkComponentImage::OnPaint(SkCanvas *canvas) {
     /*Draw Image Shadow*/
     if(contentShadow) {
       if(imageProps.resizeMode == ImageResizeMode::Repeat) {
-      /*TODO Not applied any shadow for solid Image when resizeMode is "Repeat"*/
-        sk_sp<SkImageFilter> imageFilter(SkImageFilters::Tile(targetRect,frameRect,layer()->shadowFilter));
-        shadowPaint.setImageFilter(std::move(imageFilter));
-      } else {
-        shadowPaint.setImageFilter(layer()->shadowFilter);
+        imageFilter = SkImageFilters::Tile(targetRect,frameRect,layer()->shadowFilter);
+      }
+      if(imageProps.blurRadius) {
+        imageFilter = imageFilter ? SkImageFilters::Blur(imageProps.blurRadius, imageProps.blurRadius,imageFilter) : SkImageFilters::Blur(imageProps.blurRadius, imageProps.blurRadius,layer()->shadowFilter);
       }
+      imageFilter ? shadowPaint.setImageFilter(std::move(imageFilter)) : shadowPaint.setImageFilter(layer()->shadowFilter);
```

Comment:
Can we change set filter filter based on if/else logic. if(imageFilter) set ImageFilter, elseif(shadow filter) set shadowfilter.
 Which avoid setting of nullfilter.